test/IDE/complete_at_top_level.swift

```diff
@@ -471,10 +471,10 @@ var stringInterp = "\(#^STRING_INTERP_3^#)"
 _ = "" + "\(#^STRING_INTERP_4^#)" + ""
 // STRING_INTERP: Begin completions
 // STRING_INTERP-DAG: Decl[InstanceMethod]/CurrNominal/IsSystem: ['(']{#(value): T#}[')'][#Void#];
-// STRING_INTERP-DAG: Decl[Struct]/CurrModule: FooStruct[#FooStruct#];
+// STRING_INTERP-DAG: Decl[GlobalVar]/Local: fooObject[#FooStruct#]; name=fooObject
```

Comment:
The old line here tests that we're suggesting a type name; the new one tests that we're suggesting a global variable. If anything, I would expect this PR to include the exact opposite diff. Is this a regression in code completion behavior, or is the test change a mistake, or is there something else going on that I'm missing?

Reply:
I'm actually not sure if this test is just reflecting an arbitrary choice that has changed due to new overloads, or whether something is wrong. @rintaro @benlangmuir  any idea whether the completion is still exhibiting correct behavior here?

Reply:
This is a test case that checks all global symbols are suggested. Both 'FooStruct' and 'fooObject' should be suggested here.
Could you check the actual result if it includes 'FooStruct'?

Reply:
see updated PR comment [here](https://github.com/apple/swift/pull/32113#discussion_r445905077)